SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -131,15 +132,18 @@ void USpatialGameInstance::StartSpatialConnection()
 		TryInjectSpatialLocatorIntoCommandLine();
 		CreateNewSpatialConnectionManager();
 	}
-#if TRACE_LIB_ACTIVE
 	else
 	{
 		// In native, setup worker name here as we don't get a HandleOnConnected() callback
-		FString WorkerName =
-			FString::Printf(TEXT("%s:%s"), *SpatialWorkerType.ToString(), *FGuid::NewGuid().ToString(EGuidFormats::Digits));
+		FString WorkerName = FString::Printf(TEXT("%s%s"), *SpatialWorkerType.ToString(), *FGuid::NewGuid().ToString(EGuidFormats::Digits));
```

Comment:
Had to remove the : here because event tracer attempts to create a file with this name

Reply:
```suggestion
		const FString fileName = FString::Printf(TEXT("%s%s");
		FString WorkerName = FString, *SpatialWorkerType.ToString(), *FGuid::NewGuid().ToString(EGuidFormats::Digits));
```
Could make it self evident with variable naming? Or conversely mention this in a comment?